app/src/theme/utils.ts

```diff
@@ -1,11 +1,142 @@
 import { getLogger } from '../util/logger'
 
-import theme from './index'
-
 const logger = getLogger('Theme::Utils')
 
+const outcomes = {
```

Comment:
Looks like the outcome colors aren't getting imported from the right spot now that they've moved
![image](https://user-images.githubusercontent.com/30579067/125837269-4c6d43f4-fdfb-4cec-9f13-1374c4e4d334.png)